src/connections/allContacts.js

```diff
@@ -1,5 +1,62 @@
 import { connect, withMutations } from 'cozy-client'
 import flow from 'lodash/flow'
+import array from 'lodash/array'
+import isEqual from 'lodash/isEqual'
+
+function flatten(arrayOfArrays) {
```

Comment:
yes, there is an addiction we could add to avoid this function of a line. I chose not to add it as an dependency, but I will follow your recommendation.